docs/user/quick-start.md

```diff
@@ -0,0 +1,80 @@
+# Quick Start
```

Comment:
I think you should remove the quick start from the main README.md then as this here is much better imo. Additionally, I think the local start guide with the watcher has big overlaps so that should be unified.

Reply:
Can you align with @jeremyharisch when to officially enable watcher in our control plane? then we need to unify these guides asap

Reply:
confirmed with @janmedrek , the watcher is not planed to introduced into current document, until we released it, the document can be revisited later.